docs/ops/reduction/ReduceL2_4.md

```diff
@@ -0,0 +1,88 @@
+## ReduceL2 <a name="ReduceL2"></a>
```

Comment:
Consider adding input scalar "p" specifying the norm value. Possible values for "p" are: integer "1" or "2".

Reply:
Done.